crates/biome_css_analyze/src/keywords.rs

```diff
@@ -4958,14 +4958,77 @@ pub const KNOWN_US_BROWSER_PROPERTIES: [&str; 517] = [
     "zoom",
 ];
 
+pub const MEDIA_FEATURE_NAMES: [&str; 60] = [
+    "any-hover",
+    "any-pointer",
+    "aspect-ratio",
+    "color",
+    "color-gamut",
+    "color-index",
+    "device-aspect-ratio",
+    "device-height",
+    "device-posture",
+    "device-width",
+    "display-mode",
+    "dynamic-range",
+    "environment-blending",
+    "forced-colors",
+    "grid",
+    "height",
+    "horizontal-viewport-segments",
+    "hover",
+    "inverted-colors",
+    "light-level",
+    "max-aspect-ratio",
+    "max-color",
+    "max-color-index",
+    "max-device-aspect-ratio",
+    "max-device-height",
+    "max-device-width",
+    "max-height",
+    "max-monochrome",
+    "max-resolution",
+    "max-width",
+    "min-aspect-ratio",
+    "min-color",
+    "min-color-index",
+    "min-device-aspect-ratio",
+    "min-device-height",
+    "min-device-width",
+    "min-height",
+    "min-monochrome",
+    "min-resolution",
+    "min-width",
+    "monochrome",
+    "nav-controls",
+    "orientation",
+    "overflow-block",
+    "overflow-inline",
+    "pointer",
+    "prefers-color-scheme",
+    "prefers-contrast",
+    "prefers-reduced-data",
+    "prefers-reduced-motion",
+    "prefers-reduded-transparency",
+    "resolution",
+    "scan",
+    "screen-spanning",
+    "scripting",
+    "update",
+    "vertical-viewport-segments",
+    "video-color-gamut",
+    "video-dynamic-range",
+    "width",
+];
+
 #[cfg(test)]
 mod tests {
     use std::collections::HashSet;
 
     use super::{
         FUNCTION_KEYWORDS, KNOWN_EDGE_PROPERTIES, KNOWN_EXPLORER_PROPERTIES,
         KNOWN_FIREFOX_PROPERTIES, KNOWN_PROPERTIES, KNOWN_SAFARI_PROPERTIES,
-        KNOWN_SUMSUNG_INTERNET_PROPERTIES, KNOWN_US_BROWSER_PROPERTIES,
+        KNOWN_SUMSUNG_INTERNET_PROPERTIES, KNOWN_US_BROWSER_PROPERTIES, MEDIA_FEATURE_NAMES,
```

Comment:
off-topic

Should it be `KNOWN_SAMSUNG_INTERNET_PROPERTIES` instead?

Reply:
Thanks, I fixed.